src/CQRS/LeanCode.CQRS.MassTransitRelay/ConsumerConfigurationExtensions.cs

```diff
@@ -0,0 +1,36 @@
+using MassTransit;
+
+namespace Leancode.CQRS.MassTransitRelay;
+
+public static class ConsumerConfigurationExtensions
+{
+    public static ISendPipeConfigurator ConfigureSendActorIdPropagation(ISendPipeConfigurator cfg)
+    {
+        cfg.UseExecute(ex =>
+        {
+            if (
+                ex.TryGetPayload<ConsumeContext>(out var payload)
+                && payload.TryGetHeader<string>("ActorId", out var actorId)
```

Comment:
The header name should be put in `const` somwehere, because now it is used in... 8 places.